src/applications/gi/containers/LandingPage.jsx

```diff
@@ -67,7 +68,15 @@ export class LandingPage extends React.Component {
       }
     });
 
-    this.props.router.push({ pathname: 'search', query });
+    // prod flag for CT-116 - #19864
+    if (environment.isProduction()) {
+      this.props.router.push({ pathname: 'search', query });
+    } else if (isVetTecSelected(this.props.filters)) {
+      delete query.vetTecProvider;
+      this.props.router.push({ pathname: 'program-search', query });
+    } else {
+      this.props.router.push({ pathname: 'search', query });
+    }
```

Comment:
Since the if and the else have the same result, I prefer to have them in one condition. If it's just to make prod flag removal easier, I'm fine with that.